gcloud-java-core/src/main/java/com/google/gcloud/ServiceOptions.java

```diff
@@ -150,6 +155,16 @@ public B serviceRpcFactory(ServiceRpcFactory<ServiceRpcT, OptionsT> serviceRpcFa
       this.serviceRpcFactory = serviceRpcFactory;
       return self();
     }
+
+    public B connectTimeout(int connectTimeout) {
```

Comment:
I was planning to add some docs for all the methods in the builder, we can make it part of this PR if you don't mind.